### PR TITLE
Users and Groups Data Functions created

### PR DESCRIPTION
**Pull request by YashBharambay**
Modified the address object

### CODE REVIEW

**Review by abhishekkocharekar — CHANGES_REQUESTED** [review]
City, state and street address have to be inside the address object.

File: data/users.js [hunk 1]
```diff
@@ -1,124 +1,121 @@
...
+const updateUser = async (
+  userId,
+  password,
+  firstName,
+  lastName,
+  email,
+  gender,
+  userDescription,
+  profileImgUrl,
+  address,
```
Comment (abhishekkocharekar) [inline data/users.js:78]: Do the same thing in updateUser too. Also, We won't be getting address as an input parameter.

**Review by abhishekkocharekar — CHANGES_REQUESTED** [review]

**Review by prathyekareddy — CHANGES_REQUESTED** [review]
Reverting

**Comment by prathyekareddy** [comment]
Reverting

**Review by prathyekareddy — COMMENTED** [review]
Reverting

**Comment by prathyekareddy** [comment]
Reverted

**Review by prathyekareddy — COMMENTED** [review]
Nice work!

**Comment by prathyekareddy** [comment]
Groups Data file updated- 11/19

**Review by prathyekareddy — COMMENTED** [review]
Reverting

**Comment by prathyekareddy** [comment]
Reverting

**Review by prathyekareddy — APPROVED** [review]
Merged - 11/19

**Review by abhishekkocharekar — APPROVED** [review]
Looks Good!

**Comment by sabahahmed1** [comment]
Nice job!

**Comment by SwarajsingPatil** [comment]
Nice work!

**Comment by SwarajsingPatil** [comment]
Nice work!

**Pull request merged**